packages/host/app/components/operator-mode/code-submode/playground/playground-content.gts

```diff
@@ -138,7 +138,7 @@ export default class PlaygroundContent extends Component<Signature> {
       }
       .format-chooser {
         position: sticky;
-        bottom: 0;
+        bottom: var(--search-sheet-closed-height);
```

Comment:
This is causing a large gap under the format-chooser in playground-preview. No need to change this.